### ISSUE TITLE
Cover Image text shows bold when bold is not selected in backend

### ISSUE THREAD

**Issue by ianbelanger79** [WordPress/twentynineteen#440]
When editing Cover Image text, it shows as bold in the backend, while it's not bold on the frontend.  When bold is selected in the backend, it shows as not bold, while on the frontend it is bold.

Backend - Bold not selected
![bold-not-selected-text-bold-backend](https://user-images.githubusercontent.com/34601694/47808424-bd4fad00-dd14-11e8-8f46-cc5243218539.PNG)

Backend - Bold selected
![bold-selected-text-not-bold-backend](https://user-images.githubusercontent.com/34601694/47808445-cb9dc900-dd14-11e8-9c83-3e0542194a4f.PNG)

Frontend - Bold not selected
![bold-not-selected-text-not-bold-frontend](https://user-images.githubusercontent.com/34601694/47808489-deb09900-dd14-11e8-8200-a2aca7410034.PNG)

Frontend - Bold selected
![bold-selected-text-bold-frontend](https://user-images.githubusercontent.com/34601694/47808504-e53f1080-dd14-11e8-8a7d-d62b681439fe.PNG)

So basically, the text appears correctly on the frontend, but is opposite in the backend.

WP Version: 5.1-alpha-20181026.192038
GB Version: 4.3.0-alpha-3503c0d

**Comment by ianbelanger79** [comment]
Fixed via #497
